std-bits/base/src/main/java/org/enso/base/enso_cloud/audit/AuditLogApiAccess.java

```diff
@@ -1,268 +1,17 @@
-      return cachedRequestConfig;
-    }
-
-    var uri = URI.create(CloudAPI.getAPIRootURI() + "logs");
-    var config = new RequestConfig(uri, AuthenticationProvider.getAccessToken());
-    cachedRequestConfig = config;
-    return config;
-  }
-
-  /**
-   * Contains information needed to build a request to the Cloud Logs API.
-   *
-   * <p>This information must be gathered on the main Enso thread, as only there we have access to
-   * the {@link AuthenticationProvider}.
-   *
-   * <p>We associate an instance with every message to be sent. When sending multiple messages in a
-   * batch, we will use the config from one of them. This should not matter as in normal operations
-   * the configs will be the same, they only change during testing. Tests should this into account,
-   * by sending the last message in synchronous mode.
-   */
-  private record RequestConfig(URI apiUri, String accessToken) {}
-
-  private void sendLogRequest(HttpRequest request, int retryCount) throws RequestFailureException {
-    try {
-      try {
-        if (httpClient == null) {
-          httpClient = HttpClient.newBuilder().followRedirects(HttpClient.Redirect.ALWAYS).build();
-        }
-        HttpResponse<String> response =
-            httpClient.send(request, HttpResponse.BodyHandlers.ofString());
-        if (response.statusCode() < 200 || response.statusCode() >= 300) {
-          throw new RequestFailureException(
-              "Unexpected status code: " + response.statusCode() + " " + response.body(), null);
-        }
-      } catch (IOException | InterruptedException e) {
-        // Promote a checked exception to a runtime exception to simplify the code.
-        var errorMessage = e.getMessage() != null ? e.getMessage() : e.toString();
-        throw new RequestFailureException("Failed to send log messages: " + errorMessage, e);
-      }
-    } catch (RequestFailureException e) {
-      if (retryCount < 0) {
-        logger.severe("Failed to send log messages after retrying: " + e.getMessage());
-        throw e;
-      } else {
-        logger.warning("Exception when sending log messages: " + e.getMessage() + ". Retrying...");
-        sendLogRequest(request, retryCount - 1);
-      }
-    }
-  }
-
-  interface LogMessage {
-    String payload();
-  }
-
-  static class RequestFailureException extends RuntimeException {
-    public RequestFailureException(String message, Throwable cause) {
-      super(message, cause);
-    }
-  }
-
-  /**
-   * A record that represents a single log to be sent.
-   *
-   * <p>It may contain the `completionNotification` future that will be completed when the log is
-   * sent. If no-one is listening for confirmation, that field will be `null`.
-   */
-  record LogJob(
-      LogMessage message,
-      CompletableFuture<Void> completionNotification,
-      RequestConfig requestConfig) {}
-
-  void resetCache() {
-    cachedRequestConfig = null;
+  @Override
+  public URI endpoint() {
```

Comment:
According to [clarity of access modifiers](http://wiki.apidesign.org/wiki/ClarityOfAccessModifiers), the method should be `protected` and the method it overrides should be `protected abstract`.